backend/src/app/db.clj

```diff
@@ -186,17 +184,17 @@
               (sql/insert table params opts)
               (assoc opts :return-keys true))))
 
-(defn- select-values [map ks]
-  (reduce #(conj %1 (map %2)) [] ks))
+;; (defn- select-values [map ks]
+;;   (reduce #(conj %1 (map %2)) [] ks))
 
 (defn insert-multi!
   [ds table param-list]
   (doseq [params param-list]
-      (insert! ds table params))
+    (insert! ds table params))
   ;; FIXME: Won't work
   #_(let [keys (->> param-list first keys (into []))
-        params (->> param-list (mapv #(->> keys (select-values %) (into []))) )]
-    (jdbc-sql/insert-multi! ds table keys params default-options)))
+          params (->> param-list (mapv #(->> keys (select-values %) (into []))) )]
```

Comment:
Don't use the clojure one. They do different things. The currently used if a key is not present in a map will return nil for that key. Clojure removes that value. Is necesary otherwise will fail because could have different number of rows.

Reply:
This change only implies indentation change, no behavior/code is changed.